### DISCUSSION TITLE
Stable Diffusion功能不能使用？

### DISCUSSION THREAD

**Discussion by huajun999** [anse-app/anse#2] [Q&A] [answered]
Stable Diffusion功能不能使用，我填入的是官方https://platform.stability.ai/的api key。返回
404
The requested resource could not be found..请教一下各位大神，老哥们。是哪里的原因？

**Comment by ddiu8081** [comment, marked as answer]
We are temporarily using https://replicate.com/ as the provider of the Stable Diffusion interface, so you will need to fill in the API key of the Replicate website. 

Other platforms will be supported later.
